### DISCUSSION TITLE
请求状态码200但无返回数据

### DISCUSSION THREAD

**Discussion by Geek-zhihao** [Johnserf-Seed/TikTokDownload#670] [Q&A]
想请教一下各位大佬，调用了F2中生成msToken和X-Bogus的接口生成了请求URL，请求返回的状态码是200但是没有响应数据可能是什么原因呢？
![屏幕截图 2024-03-02 002248](https://github.com/Johnserf-Seed/TikTokDownload/assets/139238643/967a9a8d-c10b-458f-b953-0aeb8b22ba7f)

**Comment by Johnserf-Seed** [comment]
更换cookie

**Reply by Geek-zhihao** [reply]
我的测试代码换Cookie后还是无数据，可能是请求传参的问题，暂时还没找到原因

**Comment by Johnserf-Seed** [comment]
https://johnserf-seed.github.io/f2/guide/apps/douyin/#%E5%8D%95%E4%B8%AA%E4%BD%9C%E5%93%81%E6%95%B0%E6%8D%AE-%F0%9F%9F%A2
https://johnserf-seed.github.io/f2/guide/apps/douyin/#%E4%BD%BF%E7%94%A8%E6%8E%A5%E5%8F%A3%E6%A8%A1%E5%9E%8B%E7%94%9F%E6%88%90xb%E5%8F%82%E6%95%B0-%F0%9F%9F%A2
可以直接使用示例代码

**Reply by Geek-zhihao** [reply]
示例代码带无登录状态的Cookie能返回数据
